src/main/scala/io/iohk/ethereum/network/rlpx/RLPxConnectionHandler.scala

```diff
@@ -138,7 +138,7 @@ class RLPxConnectionHandler(
         context.parent ! MessageReceived(message)
 
       case Failure(ex) =>
-        log.error(ex, s"Cannot decode message from $peerId")
```

Comment:
Can we do an `if debugEnabled` so as not to lose the trace when the LEVEL is `debug`?